src/Pages/LogIn/LoginScreen.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import LoginTooltipIcon from "./LoginTooltipIcon";
+import LoginPageLoginBtn from "./LoginPageLoginBtn";
+import CreateAccountBtn from "./CreateAccountBtn";
+import LineContainer from "./LineContainer";
+
+class LoginScreen extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      inputType: "password",
+      visible: "표시",
+    };
+  }
+
+  userInfo = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handlePassword = () => {
+    this.setState({
+      inputType: this.state.inputType === "password" ? "text" : "password",
+      visible: this.state.visible === "표시" ? "숨김" : "표시",
+    });
+  };
+
+  tryLogin = (e) => {
+    e.preventDefault();
+    fetch("http://10.58.1.69:8000/user/signin", {
```

Comment:
```suggestion
    fetch(`${API}/user/signin`, {
```
주소 등은 src 폴더 아래 config.js 파일 만들고 그곳에 모아서 관리해주세요

Reply:
적용시켰습니다. 이렇게 또 새로운 공부 하게 해주셔서 감사합니다~~

Reply:
200oK
API 주소는 웬만하면 동일하고 하나의 프로젝트 폴더의 여러 파일에서 fetch 를 실행시키기 때문에 API 주소가 바뀌게 되면 전부 바꾸는 것은 귀찮은 작업이기 때문에 하나만 바꾸면 되게 따로 관리한다.